docs/quickstart.md

```diff
@@ -3,13 +3,22 @@ import TabItem from '@theme/TabItem';
 import IframeVideo from '@site/src/components/IframeVideo/index';
 import LinkButton from '@site/src/components/LinkButton/index';
 import SdkInstallation from './\_sdk-installation.mdx';
+import ThemedImage from '@theme/ThemedImage';
 
 # Blind App Quickstart
 
 ## What is a blind app?
 
 A blind app runs blind computation on Nillion using one or more privacy-preserving [Nada](/nada-lang) programs. Nada programs compute on secret integers without ever seeing the underlying input values, making them ideal for sensitive data operations.
 
+<ThemedImage
+  alt="Nillion Quickstart Demo"
+  sources={{
+    light: '/img/nillion_quickstart_demo_light.png',
+    dark: '/img/nillion_quickstart_demo_dark.png',
+  }}
+/>
+
 ## Start building blind apps on Nillion
```

Comment:
I noticed step 3 in this section still references cra-nillion: https://nillion-docs-git-feat-js-client-release-nillion.vercel.app/quickstart#start-building-blind-apps-on-nillion

Update this text + link to create-nillion-app

> 3. [Build a blind app](/quickstart-blind-app) with the [cra-nillion](https://github.com/NillionNetwork/cra-nillion) starter repo (based on Create React App) and run your app locally on the [nillion-devnet](/nillion-devnet)